src/wrappers/variant_search.py

```diff
@@ -0,0 +1,72 @@
+import itertools
+import subprocess
+import tempfile
+from os import environ
+
+from pyfasta import Fasta
+
+from wrappers.cobs import COBS_TERM_SIZE
+
+TB_REF = environ.get('TB_REF', '/data/NC_000962.3.fasta')
+
+
+class VariantSearch:
+    def __init__(self, cobs, reference_path=TB_REF):
+        self.cobs = cobs
+        self.reference_path = reference_path
+
+    def search(self, ref_base, pos, alt_base):
+        var_name = "".join([ref_base, str(pos), alt_base])
+        fasta_string = self.create_variant_probe_set(var_name=var_name)
+        with tempfile.NamedTemporaryFile() as fp:
```

Comment:
Can we try `tempfile.SpooledTemoraryFile`? Try to avoid disk access for reasonably short string?

Reply:
Sorry, forget about this comment. It seems that `tempfile.SpooledTemporaryFile` won't work in our case.